graylog2-web-interface/src/views/logic/queries/SearchType.ts

```diff
@@ -36,7 +36,7 @@ type Interval = AutoInterval | TimeUnitInterval;
 
 type SearchTypePivot = {
   type: string;
-  field: string;
+  fields: Array<string>;
```

Comment:
I am not sure if the structure changed over time, but looking at the actual backend response the `fields` attribute is always an array.